src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/InterchangeHeader.java

```diff
@@ -17,11 +18,11 @@
 @Getter @Setter @RequiredArgsConstructor
 public class InterchangeHeader extends Segment {
 
-    private static DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("yyMMdd:HHmm");
+    private static DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("yyMMdd:HHmm").withZone(ZoneOffset.UTC);
```

Comment:
`final` is missing ?

Reply:
Commented above, should use "Europe/London" time zone and we should add unit tests around BST/GMT when working with EDIFACT or FHIR timestamps.